pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
     "pulumiverse-heroku>=1.0.3,<2",
     "pulumi-eks>=3.8.0,<4",
     "pulumi-aws-native>=1.25.0,<2",
+    "pyright>=1.1.394",
```

Comment:
Why are we adding pyright to the dependencies?

Reply:
Oops. Fixing. Thanks!